bin/varnishd/mgt/mgt_main.c

```diff
@@ -158,6 +158,9 @@ usage(void)
 	printf(FMT, "-j jail[,options]", "Jail specification");
 #ifdef HAVE_SETPPRIV
 	printf(FMT, "", "  -j solaris");
+#endif
+#ifdef WITH_LINUX_JAIL
+	printf(FMT, "", "  -j linux_experimental");
 #endif
```

Comment:
For consistency I'd rather see something like `HAVE_SECCOMP` or something similar instead of `WITH_LINUX_JAIL`.

That also means changes in `configure.ac` which I promised I'd help make.

Reply:
With the current `AC_CHECK_LIB` in `configure.ac` we should already have a macro called `HAVE_LIBSECCOMP` out of the box!